store/cafs/src/addFilesFromDir.ts

```diff
@@ -41,7 +41,7 @@ async function _retrieveFileIntegrities (
 ) {
   try {
     const files = await fs.readdir(currDir)
-    await Promise.all(files.map(async (file) => {
+    await Promise.allSettled(files.map(async (file) => {
```

Comment:
This is not a good solution because you just silence any type of issues.

Reply:
In that case, I'd argue it's better UX to propagate the error and fail the installation if something unexpected happens. 

Right now, the index is returned with just a subset of what actually should be populated.